Sources/LeafKit/LeafLexer.swift

```diff
@@ -1,147 +1,210 @@
-struct LeafLexer {
-    private var buffer: ByteBuffer
+extension ByteBuffer {
+    mutating func stringify() -> String? {
```

Comment:
Prefixing this with `read...` is NIO's pattern for letting you know it's a mutating method (increments the reader index). Maybe `readRestAsString()` or `readEntireString()`?

Non-mutating methods are `get...` (and you pass in an explicit reader index).